src/components/DataInput/CheckboxGroupPopup.tsx

```diff
@@ -43,7 +43,7 @@ const CheckboxGroupPopup: React.FC<Props> = (props) => {
                 onClick={(event) => setPopoverAnchorElement(event.currentTarget)}
                 disabled={!props.labelsAndKeys.length}
                 type="button"
-                id="status-button"
+                id="checkbox-status-button"
```

Comment:
This component is used in multiple places so I don't think setting this to a hard-coded id is a good idea.

If we don't actually need the id, can we remove it? (this seems to be the case?)

If we do need the ID for this to work, can we change this so that we have some sort of dynamic id name?

Reply:
sure, I think you're right, we don't need it